transcom/mymove
# B 17852 investigate restricting records by prime simulator

## [Agility ticket](https://www13.v1host.com/USTRANSCOM38/TeamRoom.mvc/Show/848240)

## Summary

Prime Simulator is pulling too many records and is bogged down. Added filter on API call to pull only recent moves.
Urgent change requested by Dan Schuster

These are to be checked by the author.


## Verification Steps for Reviewers

These are to be checked by a reviewer.


### Setup to Run the Code

- [Instructions for starting storybook](https://transcom.github.io/mymove-docs/docs/frontend/setup/storybook)
- [Instructions for starting the MilMove application](https://transcom.github.io/mymove-docs/docs/getting-started/application-setup/)
- [Instructions for running tests](https://transcom.github.io/mymove-docs/docs/getting-started/development/testing)

### How to test

1. Access the office app
2. Login as a prime simulator
3. results are filtered by todays date by default
4. enter an earlier date YYYY-MM-DD and confirm earlier moves are present.

### Frontend
